### DISCUSSION TITLE
Use tempdir for quarto preview instead of current workspace

### DISCUSSION THREAD

**Discussion by jooyoungseo** [quarto-dev/quarto-cli#3453] [Q&A]
From a screen-reader's viewpoint, html output that `quarto preview` produces is much more accessible when reviewing Jupyter Notebook (*.ipynb) files as opposed to conventional Jupyter Lab and VSCode interfaces. However, when I collaborate with other folks who do not use Quarto, I need to manually delete the preview html file and dependencies after each time I finish reviewing Notebook content.

Would there be any way to use temp directory to serve preview files instead of the current working directory?

**Comment by cscheid** [comment]
We do have an `output-dir` option, but I believe it'll only work in website settings. Would it be possible to change where Jupyter lab or VS Code render instead?
